src/main/resources/data.sql

```diff
@@ -0,0 +1,27 @@
+DROP TABLE IF EXISTS persons;
+DROP TABLE IF EXISTS firestations;
+DROP TABLE IF EXISTS medicalrecords;
+
+CREATE TABLE persons (
```

Comment:
Hmm I think that a table name should be singular -> `PERSON`

Reply:
yes you're right